### PR TITLE
Removed other internal domains

### PR DESCRIPTION
**Pull request by aarongilbert123**

### CODE REVIEW

File: Chef/SetupWorkstation.ps1 [hunk 1]
```diff
@@ -63,5 +63,5 @@ vagrant plugin install vagrant-berkshelf
 
 # Add DevVm base machine as Vagrant box image
 Add-DevVm-Base-Machine-As-Vagrant-Box-Image# Add Vagrant box image
-vagrant box add "DevVmBaseImage" "\\kcura.corp\shares\Development\DevEx\DevVm_Install_Files\DevVmBase.box"
+vagrant box add "DevVmBaseImage" $Env:devvm_box_image_network_path
```
Comment (calimetikcura) [inline Chef/SetupWorkstation.ps1:66]: Where is this environment variable setup? on our server?

**Review by calimetikcura — CHANGES_REQUESTED** [review]

File: Chef/SetupWorkstation.ps1 (hunk 1 above)
Comment (aarongilbert123) [reply Chef/SetupWorkstation.ps1:66]: @calimetikcura Yes on the server

Comment (calimetikcura) [reply Chef/SetupWorkstation.ps1:66]: Have we documented it somewhere?
If not I would just add a comment at the end of this line saying to setup this env variable for this script to work.

Comment (aarongilbert123) [reply Chef/SetupWorkstation.ps1]: Done

**Review by calimetikcura — APPROVED** [review]

**Review by jake-martin — APPROVED** [review]

**Pull request merged**